frontend/test/containers/containers.redux.spec.ts

```diff
@@ -15,8 +15,7 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { INITIAL_STATE, reducer as containersReducer, ContainersActions } from '@/v5/store/containers/containers.redux';
-import { IContainersState } from '@/v5/store/containers/containers.types';
+import { INITIAL_STATE, reducer as containersReducer, ContainersActions, IContainersState } from '@/v5/store/containers/containers.redux';
```

Comment:
Is there a reason why we name each individual reducer as "reducer" when whenever we seem to import it end up renaming it anyway?

Reply:
The reason is that I continued to use the style of v4, but I changed it.